components/Buttons/src/Theming/MDCButton+MaterialTheming.m

```diff
@@ -186,4 +188,19 @@ - (void)resetButtonColorsForAllStates {
   }
 }
 
+- (void)applyTypographyScheme:(id<MDCTypographyScheming>)typographyScheme {
+  if (typographyScheme.mdc_adjustsFontForContentSizeCategory) {
```

Comment:
This can be updated to use `useCurrentContentSizeCategoryWhenApplied` now.

Until we delete `mdc_adjustsFontForContentSizeCategory` this check may have to be something like so:

```objc
BOOL useCurrentContentSizeCategoryWhenApplied = NO;
if ([typographyScheme respondsToSelector(useCurrentContentSizeCategoryWhenApplied)]) {
  useCurrentContentSizeCategoryWhenApplied =
      typographyScheme.useCurrentContentSizeCategoryWhenApplied;
} else {
  useCurrentContentSizeCategoryWhenApplied =
       typographyScheme.mdc_adjustsFontForContentSizeCategory;
}
```